docs/getting-started/3-load-ruleset.md

```diff
@@ -0,0 +1,43 @@
+# Load a Ruleset
```

Comment:
This is somewhat misleading - we don't describe the load process here.
The JS API is not covered here too, so feels like we should either move it back to the initial spot or explain how a ruleset can be loaded programmatically when using JS API.